Makefile

```diff
@@ -1,13 +1,15 @@
 VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
 COMMIT  := $(shell git log -1 --format='%H')
+SDKCOMMIT := $(shell echo $(shell go list -m -u github.com/cosmos/cosmos-sdk) | sed 's~github.com/cosmos/cosmos-sdk ~~')
```

Comment:
```suggestion
SDKCOMMIT := $(shell go list -m -u -f '{{.Version}}' github.com/cosmos/cosmos-sdk)
```

Reply:
Forgot to pull this in on this PR. Thank you for adding this. spent a while looking for exactly this. Going to tag you on the commit where I put it in. Thank you @alessio!